assets/src/dashboard/components/button/index.js

```diff
@@ -26,14 +26,18 @@ import styled from 'styled-components';
 import { BUTTON_TYPES, KEYBOARD_USER_SELECTOR } from '../../constants';
 
 const StyledButton = styled.button`
+  font-family: ${({ theme }) => theme.fonts.button.family};
+  font-size: ${({ theme }) => theme.fonts.button.size};
+  font-weight: ${({ theme }) => theme.fonts.button.weight};
+  line-height: ${({ theme }) => theme.fonts.button.lineHeight};
   align-items: center;
   color: ${({ theme }) => theme.colors.white};
   cursor: pointer;
   border: 1px solid transparent;
   border-radius: ${({ theme }) => theme.border.buttonRadius};
   display: flex;
   height: 40px;
-  min-width: 132px;
+  min-width: 100px;
   opacity: 0.75;
   padding: 0;
```

Comment:
With the smaller min width (which makes sense) - should there be default padding here to prevent this? 
<img width="149" alt="Screen Shot 2020-04-09 at 4 10 57 PM" src="https://user-images.githubusercontent.com/10720454/78948331-c2be3480-7a7c-11ea-9d3e-9ae233034dfa.png">

Reply:
Oooooh, I totally forgot to check the story book.  Good call, I'll fix.

Hmmm... yeah, maybe a small default padding just so long text buttons (which I would say "Default Button Demo") falls into won't smush against the corners.

Reply:
I added left/right padding of 10px.  It fixes the case shown in story book _and_ the uses of the button in the dashboard still work:
![image](https://user-images.githubusercontent.com/40646372/78949646-fd29d080-7a80-11ea-9646-cbc8221f62e2.png)